📖adding a note in deploy-image plugin doc (#2854)
kubernetes-sigs/kubebuilder: docs/book/src/plugins/deploy-image-plugin-v1-alpha.md

kubebuilder create api --group example.com --version v1alpha1 --kind Memcached --image=memcached:1.6.15-alpine --image-container-command="memcached,-m=64,modern,-v" --image-container-port="11211" --run-as-user="1001" --plugins="deploy-image/v1-alpha"
```

> <aside class="note">

Reviewer: ```suggestion
<aside class="warning">

```

> <h1> Running operator locally </h1>

Reviewer: ```suggestion
<h1> Testing locally your project</h1>

```

> For running an operator made with deploy-image plugin locally, the envvar added in `config/manager/manager.yaml` which stores the value of the image needs to be exported before running the operator locally or out of the cluster.

Reviewer: ```suggestion

The `make run` will execute the `main.go` outside of the cluster to let you test the project running it locally. Note that by using this plugin the Operand image informed will be stored via an environment variable in the  `config/manager/manager.yaml` manifest. 

Therefore, before run `make run` you need to export any environment variable that you might have. Example:

```sh
export MEMCACHED_IMAGE="memcached:1.4.36-alpine"
```
```
